docs/reference/content/tutorials/text-search.md

```diff
@@ -10,6 +10,8 @@ title = "Text Search"
 
 # Text Search
 
+> [Atlas Search](https://www.mongodb.com/cloud/atlas?tck=docs_server) makes it easy to build fast, relevance-based search capabilities on top of your MongoDB data. Try it today on [MongoDB Atlas](https://www.mongodb.com/cloud/atlas?tck=docs_server), our fully managed database as a service.
```

Comment:
I don't think this is the right link, the associated DRIVERS ticket says to link to `https://docs.atlas.mongodb.com/atlas-search/`

Reply:
fixed!

Reply:
@reggi can you make sure to push the changes to this PR? I'll update the change on the new Node.js driver docs in this PR once you've done that, thanks:
https://github.com/mongodb/docs-node/pull/112/files#diff-eb57093533a79d809482414db9cb8d3fR3-R6